tests/harness/model/deployment.py

```diff
@@ -51,7 +51,7 @@ class Config:
         """Pydantic configuration class."""
 
         validate_assignment = True
-        extra = Extra.allow
+        extra = "allow"
```

Comment:
Why is this change necessary?

Reply:
I think `extra` requires a str instead of enum. that change was actually responsible for fixing some of `mypy` errors

Reply:
Or maybe I changed something else before this and I thought it was this causing those issues haha😅